src/commcare_cloud/ansible/roles/barman/tasks/install.yml

```diff
@@ -0,0 +1,41 @@
+- name: Make sure the CA certificates are available
+  apt:
+    pkg: ca-certificates
+    state: present
+
+- name: Add PostreSQL apt repository
+  apt_repository: repo='deb http://apt.postgresql.org/pub/repos/apt/ {{ ansible_lsb.codename }}-pgdg main' state=present
+  register: add_postgres_repo
+
+- name: Add PosgreSQL apt key
+  apt_key: url=https://www.postgresql.org/media/keys/ACCC4CF8.asc state=present
```

Comment:
Can we not rely on the `common` role to do these and install the postgres client?